common/lib/api-client/transformers/move.transformer.test.js

```diff
@@ -1,3 +1,5 @@
+const { expect } = require('chai')
```

Comment:
Isn't this already globally available?

Reply:
GAH!  VSC inserting it automagickally again